platform/ui/src/components/SwitchButton/SwitchButton.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useCallback, useState } from 'react';
+
+import './switchButton.css';
+
+export enum SwitchLabelLocation {
+  left,
+  right,
+}
+
+export type SwitchButtonProps = {
+  isSwitchOn?: boolean;
```

Comment:
enabled? instead of isSwitchOn?

Reply:
enabled to me is confusing, because the opposite of enabled is disabled which to me means the button CANNOT even be clicked.

Reply:
How about checked similar to material ui

```
<Switch
  checked={checked}
  onChange={handleChange}
  inputProps={{ 'aria-label': 'controlled' }}
/>
```

Reply:
https://mui.com/material-ui/react-switch/